modules/epics/src/test/scala/engage/epics/TestEnumerated.scala

```diff
@@ -17,6 +17,6 @@ object TestEnumerated {
   implicit val testEq: Eq[TestEnumerated] = Eq.fromUniversalEquals
 
   implicit val testEnumerated: Enumerated[TestEnumerated] =
-    Enumerated.of[TestEnumerated](VAL0, VAL1, VAL2)
+    Enumerated.from[TestEnumerated](VAL0, VAL1, VAL2).withTag(_ => "TestEnumerated")
```

Comment:
This doesn't seem right, the tag should be unique and unchanging for each enumerated element while this `withTag` seems to give the same tag to each value

Reply:
Oh darn, I made this mistake in many places. Why was `Enumerated.of` removed?

Reply:
Because it was sort of based on reflection thus could cause problem if you changed it especially as it will soon be stored in the db
In this case it is not critical that it never changes but it should be different or the equals would fail

Reply:
See
https://github.com/gemini-hlsw/lucuma-core/pull/569

Reply:
Thanks that makes sense. I remember looking at that PR but I don't know what I was thinking when I did this 😅